packages/react-docs/src/pages/components/dropdown.js

```diff
@@ -0,0 +1,147 @@
+import React, { Component } from 'react';
+import ComponentDocs from '../../components/componentDocs';
+import PropTypes from 'prop-types';
+import {
+  Dropdown,
+  DropdownItem,
+  DropdownDirection,
+  DropdownPosition
+} from '@patternfly/react-core';
+import Example from '../../components/example';
+
+const propTypes = {
+  data: PropTypes.any.isRequired
+};
+
+class LiveDropdown extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isCollapsed: true
+    };
+    this.onToggle = this.onToggle.bind(this);
+    this.onSelect = this.onSelect.bind(this);
+  }
+
+  onToggle(isCollapsed) {
+    this.setState({
+      ...this.state,
+      isCollapsed
+    });
+  }
+
+  onSelect(event) {
+    this.setState({
+      ...this.state,
+      isCollapsed: !this.state.isCollapsed
+    });
+  }
+
+  render() {
+    const { children, ...props } = this.props;
+    const { isCollapsed } = this.state;
+    return (
+      <Dropdown
+        onToggle={this.onToggle}
+        onSelect={this.onSelect}
+        isCollapsed={isCollapsed}
+        {...props}
+      >
+        {children}
+      </Dropdown>
+    );
+  }
+}
+
+LiveDropdown.propTypes = {
+  children: PropTypes.node
+};
+
+LiveDropdown.defaultProps = {
+  children: null
+};
+
+const DropdownDocs = ({ data }) => {
+  const DropItems = () => (
+    <React.Fragment>
+      <DropdownItem>Link</DropdownItem>
+      <DropdownItem component="button">Action</DropdownItem>
+      <DropdownItem isDisabled>Disabled Link</DropdownItem>
+      <DropdownItem isDisabled component="button">
+        Disabled Action
+      </DropdownItem>
+      <DropdownItem isSeparator />
+      <DropdownItem>Separated Link</DropdownItem>
+      <DropdownItem component="button">Separated Action</DropdownItem>
+    </React.Fragment>
+  );
+  return (
+    <ComponentDocs data={data}>
+      <Example title="Dropdown">
+        <LiveDropdown title="Dropdown">
```

Comment:
A change is coming to the docs so this is probably fine, but in the future lets not get crazy with examples.  These are not easy to figure out for beginners. It saves us typing, but in the long run, it will save on questions on how to use them. Each example should be as close to real consumption as possible.

Reply:
I'll wait untill the docs get updated.